packages/picasso.js/src/core/chart-components/axis/axis-default-settings.js

```diff
@@ -199,6 +199,23 @@ const DEFAULT_CONTINUOUS_SETTINGS = {
      * @type {boolean=} */
     show: true,
   },
+  /**
+   * @typedef {object}
+   */
+  arc: {
+    /** Flag that determines if axis is radial
+     * @type {boolean=} */
+    isRadial: false,
```

Comment:
Should this be one more level up?

Reply:
Yeah, I think I agree with that.
* arc should be optional as well.
* startAngle and endAngle should have default values specified if they are optional as well.

Reply:
I am thinking that "arc" should be on the same level as "line". 
<img width="667" alt="Skärmavbild 2024-11-07 kl  10 39 01" src="https://github.com/user-attachments/assets/b7d6b0d9-2441-4654-9067-0102f0dcadc0">

Reply:
Yeah, that I agree on, keep arc where you have it now, but move `isRadial` out of the `arc` object, one level up that is.